plonky2x/core/src/frontend/builder/boolean.rs

```diff
@@ -1,16 +1,17 @@
 use crate::backend::circuit::PlonkParameters;
 use crate::frontend::builder::CircuitBuilder;
 use crate::frontend::vars::{BoolVariable, Variable};
+use crate::prelude::CircuitVariable;
 
 impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
     pub fn _false(&mut self) -> BoolVariable {
         let zero = self.zero::<Variable>();
-        zero.into()
+        BoolVariable::from_variables_unsafe(&[zero])
```

Comment:
Maybe we should have a convention that every time `unsafe` is used, there is a brief comment as to why that's okay? (even to make life easier for our auditors)

Reply:
Ya, makes sense.